ospo-book/content/en/03-chapter.md

```diff
@@ -173,13 +177,13 @@ There is a lack of consistency in how open source understanding and value is per
 
 `📚 Continue Here`
 
-* [FINOS open source maturity model](https://www.finos.org/blog/open-source-maturity-model-launch)
-* [Porsche Open Source Website](https://opensource.porsche.com/)
-* [OSPO maturity model](https://linuxfoundation.org/tools/the-evolution-of-the-open-source-program-office-ospo/)
-* [OSPO 101 training module - OSPO and your organization](https://github.com/todogroup/ospo-career-path/tree/main/OSPO-101/module3)
-* [Setting up an open source strategy](https://todogroup.org/resources/guides/setting-an-open-source-strategy/).
-* [Participating in open source communities](https://todogroup.org/resources/guides/participating-in-open-source-communities/)
-* [A guide to outbound open source software](https://todogroup.org/resources/guides/a-guide-to-outbound-open-source-software/)
-* [A deep dive into OSPO](https://www.linuxfoundation.org/research/a-deep-dive-into-open-source-program-offices)
-* [Open Source maturity model](https://www.finos.org/blog/open-source-maturity-model-launch)
+- [FINOS open source maturity model](https://www.finos.org/blog/open-source-maturity-model-launch)
+- [Porsche Open Source Website](https://opensource.porsche.com/)
+- [OSPO maturity model](https://linuxfoundation.org/tools/the-evolution-of-the-open-source-program-office-ospo/)
+- [OSPO 101 training module - OSPO and your organization](https://github.com/todogroup/ospo-career-path/tree/main/OSPO-101/module3)
+- [Setting up an open source strategy](https://todogroup.org/resources/guides/setting-an-open-source-strategy/).
+- [Participating in open source communities](https://todogroup.org/resources/guides/participating-in-open-source-communities/)
+- [A guide to outbound open source software](https://todogroup.org/resources/guides/a-guide-to-outbound-open-source-software/)
+- [A deep dive into OSPO](https://www.linuxfoundation.org/research/a-deep-dive-into-open-source-program-offices)
+- [Open Source maturity model](https://www.finos.org/blog/open-source-maturity-model-launch)
```

Comment:
This is a duplicate of line 180

Reply:
You can add a quick suggestion to remove it. With the `[±]` "Add a suggestion <ctrl+g>" button.

```suggestion
```

Reply:
thank you!